adobe/react-spectrum
# feat: 🎸 add press events to Tab

✅ Closes: [#6810](https://github.com/adobe/react-spectrum/issues/6810)

## ✅ Pull Request Checklist:


## 📝 Test Instructions:

- `yarn start`
- Navigate to Tabs Example in React Aria Components in Storybook
- Notice that clicking on a Tab fires press event actions
- Notice no breaking changes to existing tab behavior
